### DISCUSSION TITLE
Add 502 hosts

### DISCUSSION THREAD

**Discussion by papatistos** [NginxProxyManager/nginx-proxy-manager#1304] [Ideas]
It would be great have 502 hosts that work just like the existing 404 hosts. The idea is that users will see a maintenance page while a container is offline, rather than the standard "bad gateway" page.

**Comment by TheStaticTurtle** [comment]
This is not the solution, but I wrote myself a very simple docker container that display a nice static page. I then can redirect to this container in case I want to disable something

**Comment by aab12345** [comment]
> This is not the solution, but I wrote myself a very simple docker container that display a nice static page. I then can redirect to this container in case I want to disable smething

Would you mind sharing this on a GitHub please and what you did on NPM ?

**Reply by TheStaticTurtle** [reply]
It's on my [github](https://github.com/TheStaticTurtle/ImDead), pretty much everything is in the readme, as I said this isn't the best solution and might not handle high traffic loads.
